src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Shape:\t" + FigureName.CIRCLE
```

Comment:
Use name() for getting String representation of enum constants instead of relying on the toString() method which could be overridden. Replace FigureName.CIRCLE with FigureName.CIRCLE.name().